src/main/java/projectzero/cli/CliApplication.java

```diff
@@ -1,122 +1,351 @@
+                            break;
+                        case "addRelationship":
+                            addRelationships(arguments);
+                            break;
+                        case "deleteRelationship":
+                            deleteRelationship(arguments);
+                            break;
+                        case "save":
+                            try {
+                                MainManager.save(arguments);
+                                System.out.println("File saved.");
+                            } catch (Exception e) {
+                                System.out.println("Did not save correctly.");
+                            }
+                            break;
+                        case "load":
+                            try {
+                                MainManager.load(arguments);
+                                System.out.println("File Loaded.");
+                            } catch (Exception e) {
+                                System.out.println("Did not load correctly.");
+                            }
+                            break;
+
+                        case "editClass":
+                            editClass(arguments);
+                            break;
+                        case "editField":
+                            editField(arguments);
+                            break;
+                        case "editMethod":
+                            editMethod(arguments);
+                            break;
                     }
                 }
             }
-        } while (inInitMenu);
+        }
+        catch(IndexOutOfBoundsException e){
+            System.out.println("Invalid input\n");
+        }
+    }
+
+    private void editMethod(String arguments) {
+        String className = arguments.substring(0, arguments.indexOf(" "));
+        arguments = arguments.substring(className.length() + 1, arguments.length());
+        String oMethodName = arguments.substring(0, arguments.indexOf(" "));
+        String nMethodName = arguments.substring(arguments.indexOf(" ") + 1);
+        try{
+            MainManager.getUmlClass(className).updateMethod(MainManager.getUmlClass(className).getMethod(oMethodName), new Method(nMethodName));
+            System.out.println("Method " + oMethodName + " has been changed to " + nMethodName + ".");
+        }catch(InvalidNameException e){
+            System.out.println("Invalid Class name.");
+        }catch(NullPointerException e){
+            System.out.println("Method not found.");
+        }
+    }
 
+    private void editField(String arguments) {
+        String className = arguments.substring(0, arguments.indexOf(" "));
+        arguments = arguments.substring(className.length() + 1, arguments.length());
+        String oFieldName = arguments.substring(0, arguments.indexOf(" "));
+        String nFieldName = arguments.substring(arguments.indexOf(" ") + 1);
```

Comment:
This repeats in multiple methods can be put into its own method maybe. Not an issue just a suggestion for decrease repetitive code.